guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md

```diff
@@ -32,6 +32,12 @@ vendor/bin/mftf command [options] [arguments]
 
 The following is a list of the most used commands.
 
+#### Options
+
+Option | Description|
+---|---
+`-u, --upgrade`   | Upgrades existing MFTF tests according to last major release requirements. Only tests in the default location will be upgraded with this flag.
```

Comment:
@KevinBKozan, is it the same as `upgrade:tests` below?

Reply:
Yes, doing `build:project --upgrade` automatically runs the `upgrade:tests` command.